app-backend/db/src/main/resources/migrations/V75__add_index_to_annotation_labels.sql

```diff
@@ -0,0 +1,7 @@
+CREATE INDEX CONCURRENTLY IF NOT EXISTS annotation_labels_is_active_idx
```

Comment:
I kept them in separate migrations since postgres told me that transactional and un-transactional migrations cannot be run in one.

I used `CREATE INDEX CONCURRENTLY IF NOT EXISTS` on these two fields, since it seems that we don't want to block writes on the `annotation_labels` table on prod. The downside of this is specified in the [docs](https://www.postgresql.org/docs/12/sql-createindex.html#SQL-CREATEINDEX-CONCURRENTLY):
>When this option is used, PostgreSQL must perform two scans of the table, and in addition it must wait for all existing transactions that could potentially modify or use the index to terminate. Thus this method requires more total work than a standard index build and takes significantly longer to complete. However, since it allows normal operations to continue while the index is built, this method is useful for adding new indexes in a production environment. Of course, the extra CPU and I/O load imposed by the index creation might slow other operations.